README.md

```diff
@@ -1,2 +1,18 @@
 # TODO
-A TO-DO web application using Django.
+
+## A TO-DO web application using Django
+
+### prerequisite
+
```

Comment:
add 1 more file
requirement_locked.txt

this should have package + version, to do this:
pip freeze > requirement_locked.txt